src/index.ts

```diff
@@ -40,7 +40,7 @@ export * from './dex/models/dex-transaction';
 
 export * from './dex/base-dex';
 export * from './dex/minswap';
-export * from './dex/sundaeswap';
+export * from './dex/sundaeswap-v1';
 export * from './dex/muesliswap';
```

Comment:
I think we may also need to eventually add `export * from './dex/sundaeswap-v3';` here.

Reply:
Fixed this in 06982f7c416cfb0c8902efa53d870a7ae6d31b2f